arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -2186,15 +2146,34 @@
       },
       "discriminator": "objectType"
     },
-    "WorkloadProtectableItem": {
-      "description": "Base class for backup item. Workload-specific backup items are derived from this class.",
-      "type": "object",
+    "WorkloadProtectableItemResource": {
```

Comment:
I'm seeing a few linter errors -- could you use VSCode and grab the linter extension (instructions are at https://github.com/Azure/openapi-lint-extension ) and make sure that you're not getting any linter errors.

(You're missing documentation in a few places and may want to consider using `x-ms-client-flatten` on your nested properties.)